arangod/Aql/IdExecutor.h

```diff
@@ -31,6 +31,8 @@
 #include <tuple>
 #include <utility>
 
+// TODO Clean up unused versions
+
```

Comment:
unused versions of what?

Reply:
There are several variants of the IdExecutor, some of them now unused. I'll extend the comment a little, doing that will follow in the cleanup PR.